template/shared/client/app.js

```diff
@@ -13,41 +13,47 @@ var domReady = require('domready');
 
 module.exports = {
     // this is the the whole app initter
-    blastoff: function () {
-        var self = window.app = this;
-
-        // create our global 'me' object and an empty collection for our people models.
-        window.me = new Me();
-        this.people = new People();
-
-        // init our URL handlers and the history tracker
-        this.router = new Router();
+    blastoff: function() {
+        app.extend({
+
+            // create the global 'me' object and an empty collection for our people models
+            me: new Me(),
+            users: new People(),
+
+            // init our URL handlers and the history tracker
+            router: new Router(),
+
+            // This is how you navigate around the app.
+            // this gets called by a global click handler that handles
+            // all the <a> tags in the app.
+            // it expects a url without a leading slash.
+            // for example: "costello/settings".
+            navigate: function(page) {
```

Comment:
I'd kind of hoped that this method would get "promoted" into ampersand-app.  It seems like pretty much every ampersand application would need it, so it's always seemed wrong for it to have to be copy/pasted into the main app file.  Now that there's a parent, it seems like a great place to put it.